test/application.test.ts

```diff
@@ -21,8 +23,10 @@ describe('Application', () => {
     // Clear log output
     output = []
 
-    app = new Application({} as any)
-    app.auth = jest.fn().mockReturnValue({})
+    github = new GitHubApp(1, 'cert')
+    github.auth = jest.fn().mockReturnValue({})
```

Comment:
These tests would be clearer to me if we used `app.github.auth` to replace `app.auth` instead of `github.auth`.

Is there reasoning for not having it that way?